src/lib.rs

```diff
@@ -24,17 +24,17 @@
 //! [1]: ../kademlia_routing_table/index.html
 //! [2]: ../xor_name/struct.XorName.html
 //!
-//! Messages are exchanged between _authorities_, where an `Authority` can be an
-//! individual client or node, or a group of nodes. In both cases, messages are cryptographically
-//! signed by the sender, and in the latter case it is verified that a sufficient number of group
-//! members agree on the message: Only if that quorum is reached, the message is delivered. In
-//! addition, each message has a unique ID, and is delivered only once.
+//! Messages are exchanged between _authorities_, where an `Authority` can be an individual client
+//! or node, or a collection of nodes called a "section". In both cases, messages are
+//! cryptographically signed by the sender, and in the latter case it is verified that a sufficient
+//! number of section members agree on the message: only if that quorum is reached, the message is
+//! delivered. In addition, each message has a unique ID, and is delivered only once.
 //!
-//! Group authorities are also addressed using a single `XorName`. The members of that group are
+//! Section authorities are also addressed using a single `XorName`. The members of that section are
 //! the nodes that are closest to that name. Since nodes are assigned their name by the network,
-//! this provides redundancy and resilience: A node has no control over which group authorities it
-//! will be a member of, and without a majority in the group it cannot forge a message from that
-//! group.
+//! this provides redundancy and resilience: a node has no control over which section authority it
+//! will be a member of, and without a majority in the section it cannot forge a message from that
+//! section.
```

Comment:
In general documentation we should probably refer to both groups and sections? Or mention below that the same also applies to groups.